### ISSUE TITLE
Password: (accessiblility) cannot access toggle button using keyboard

### ISSUE THREAD

**Issue by arnodemer** [primefaces/primereact#6046]
### Describe the bug

focus cannot be moved to Password's toggle button, then cannot toggle using keyboard

### PrimeReact version

10.5.1

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

1. go to Password toggle show case : https://primereact.org/password/#togglemask
2. use tab to set focus on password field
3. hit tab several times, focus is never set on toggle password button

### Expected behavior

Toggle button should be able to be activated using keyboard
Focus should also stay on toggle button when password state is toggled.

**Comment by melloware** [comment]
Agreed it looks like PrimeVue has not implemented this yet either.

**Comment by akshayaqburst** [comment]
@melloware Can I work on this?

**Comment by arnodemer** [comment]
Hi ! it would be nice if we could also set a tooltip for this toggle button (password component already have one, but it's for input ).
Thanks a lot !

**Comment by melloware** [comment]
@akshayaqburst sure!

**Comment by akshayaqburst** [comment]
@melloware Created PR for review - https://github.com/primefaces/primereact/pull/6052. Thanks,

**Comment by melloware** [comment]
Some code review comments. Looking good so far!
